contrib/examples/deploy-playbook-mock.yaml

```diff
@@ -0,0 +1,58 @@
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: playbook-mock
+  labels:
+    app: playbook-mock
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: playbook-mock
+  template:
+    metadata:
+      labels:
+        app: playbook-mock
+    spec:
+      containers:
+      - name: playbook-mock
+        image: playbook-mock:canary
+        imagePullPolicy: Never
+        args:
+        - --port
+        - "8080"
```

Comment:
It looks like it doesn't log anything right now, for the purpose of verifying it might be handy to be able to see the incoming results in the pod logs. Also logrus is there if you want to get a feel for it, this is a good spot.

Reply:
Will do. I meant to do this but forgot.